### ISSUE TITLE
mAUCMLoss for multi-label classification when bs=1 loss return 0

### ISSUE THREAD

**Issue by SNEAndy** [Optimization-AI/LibAUC#37]
Hi,I'm using the mAUCMLoss to finish my multi-label classification. I realized that if the batch size of my dataloader is 1,the mAUCMLoss will always be 0,otherwise,it will not. So the batch size must greater than 1 to fit this  loss function ?

**Comment by optmai** [comment]
Do you still have this issue?
